src/main/scala/nl.knaw.dans.easy.deposit/docs/SchemedXml.scala

```diff
@@ -15,37 +15,9 @@
  */
 package nl.knaw.dans.easy.deposit.docs
 
-import better.files.StringOps
-import javax.xml.XMLConstants
-import javax.xml.transform.Source
-import javax.xml.transform.stream.StreamSource
-import javax.xml.validation.{ Schema, SchemaFactory }
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 
-import scala.util.Try
-import scala.xml.{ Elem, PrettyPrinter }
-
 trait SchemedXml extends DebugEnhancedLogging {
```

Comment:
no need for the `DebugEnhancedLogging` here, right? Same in the implementations of this trait.